src/decidim/electionguard/bulletin_board.py

```diff
@@ -46,11 +42,11 @@ def process_message(self, message_type: str, message: dict, context: Context):
         # TO-DO: verify keys?
 
         if len(context.public_keys) == context.number_of_guardians:
-            self.next_step = ProcessTrusteeElectionPartialKey()
+            self.next_step = ProcessTrusteeElectionPartialKeys()
 
 
-class ProcessTrusteeElectionPartialKey(ElectionStep):
-    message_type = 'trustee_partial_election_key'
+class ProcessTrusteeElectionPartialKeys(ElectionStep):
+    message_type = 'trustee_partial_election_keys'
```

Comment:
I've changed the subtype for this message to be consistent with the previous message (`trustee_election_keys`). It shouldn't affect anything else, but just be aware of this.